docker/ray/README.md

```diff
@@ -13,7 +13,7 @@ Images are `tagged` with the format `{Ray version}[-{Python version}][-{Platform
 | `nightly`                    | The most recent Ray development build (a recent commit from Github `master`) |
 | `6 character Git SHA prefix` | A specific development build (uses a SHA from the Github `master`, e.g. `8960af`). |
 
-The optional `Python version` tag specifies the Python version in the image. All Python versions supported by Ray are available, e.g. `py37`, `py38`, `py39` and `py310`. If unspecified, the tag points to an image using `Python 3.7`.
+The optional `Python version` tag specifies the Python version in the image. All Python versions supported by Ray are available, e.g. `py38`, `py39` and `py310`. If unspecified, the tag points to an image using `Python 3.8`.
```

Comment:
probably should add `py311` here.

Reply:
Is it fully supported? I heard there is still some dependency issue related to ml. Anyway, we can add py311 separately.